x-pack/plugins/spaces/public/nav_control/_index.scss

```diff
@@ -1,2 +0,0 @@
-@import './components/index';
-@import './nav_control';
```

Comment:
This was actually still the correct structure. Imports should happen in a single file.

Reply:
We try to follow the pattern that individual `component_name.scss` files should **not** import lower level components. Only `index.scss` should import multiple files, or a full plugin's worth of files. This helps us understand the underlying structure SASS structure without having to open each file to know what is importing what.

Yes it is correct that now we can import each component's file into it's corresponding `.tsx` file.  However, it should be using only one of these import techniques and not a combination of both:

1. One `index.scss` manifest file that imports individual component `_component.scss` files, and gets itself imported in the topmost component as `import './index.scss'`
2. Every component has a corresponding `component.scss` file that is imported directly by the component's `component.tsx` file. Meaning, lower level component SASS files like `nav_control/components/spaces_description.scss` should be imported inside `nav_control/components/spaces_description.tsx`. And not at the top level `nav_control.scss`.

I know we're in a weird transition phase right now, which is why it's all that more important that we either stick to the ole `_index.scss` manifest if you need to ensure all styles are imported by a top level component, or fully switch to the more modular import structure where each JS component imports its own SCSS file.

Reply:
@legrego I created a PR for you to show the new import structure for the modularized technique. https://github.com/legrego/kibana/pull/24

Reply:
@cchaos thanks for the explanation, that helped clarify things for me. Thanks also for the PR, that was a nice surprise!